### DISCUSSION TITLE
List of compatible Matter Devices

### DISCUSSION THREAD

**Discussion by Apollon77** [mfucci/node-matter#102] [General]
I think it is a good Idea to start to list Matter compatible Clients/Apps.

The idea is one "Post" per Device Manufactorer, so that we can also have "Device/Manufactirer dependend discussions

**Comment by Apollon77** [comment]
Because as of writing there is not that much devices out there already there is a Project to transform a ESP32 into a matter compatible device by using https://nabucasa.github.io/matter-example-apps/

@mfucci maybe easier then the Silicon Labs Dev Ket?

**Reply by mfucci** [reply]
Maybe we can try a few different devices and share the experience?

For now, I had success with the official codebase on a Raspberry Pi 4.
I didn't try to flash Silicon Labs DK yet...

**Comment by mfucci** [comment]
Thanks for this initiative.
Maybe I can create a new category for this?
So there can be one discussion per Matter compatible device.

Or we can leave everything in this discussion but it might quickly become long...

**Reply by Apollon77** [reply]
Yes own category might make most sense ...

**Reply by Apollon77** [reply]
BTW: https://csa-iot.org/csa-iot_products/?p_keywords=&p_type%5B%5D=17&p_type%5B%5D=14&p_type%5B%5D=1053&p_program_type%5B%5D=1049&p_certificate=&p_family= ... seems to be a certified devices list??

**Reply by mfucci** [reply]
Yes, tons of new devices have been added on this list in one week! There were only 6 before...

**Reply by mfucci** [reply]
Added a new category

**Comment by Apollon77** [comment]
Short here: An IT newspaper collected some upcoming updates (german) https://www.heise.de/news/Matter-Erste-Firmware-Updates-fuer-Smart-Home-Zubehoer-noch-vor-Weihnachten-7329315.html

Summary:
* Eve: plans updates for Eve Door & Window, Eve Energy, Eve Motion ( I have an Eve engery, so maybe lucky here)
* Amazon: "December 22" updates for 17 smart speakers: Echo dot Gen 3-5, Echo 3+4, Echo Show 5 and 8 and 10-3rd.Gen, Echo Flex, Echo Input, Echo Studio und Echo Plus 2nd Gen
* Signify/Philips Hue: 1st Quarter 23
* Aqara Hub M2: December 22, Gateways M1S/M1S Gen 2, Hub E1, Camera Hub G3 and Camera Hub G2H Pro "after that"
* Aquara new Thread devices planned: Door & Window Sensor P2, Motion & Light Sensor P2 in "Spring '23"
* Nuki and Nanoleaf: No updates, new devices needed, in '23

*  Amazon Alexa app as controller will start on Android "beginning '23", iOS support later

**Comment by saue0** [comment]
Is there an example for the ESP32 DEVKIt V1 using the integrated led (pin2)
I was only able to install the "All Clusters App for M5Stack ESP32 Basic Core" on my esp32, but it is not connecting.

**Reply by lboue** [reply]
> Is there an example for the ESP32 DEVKIt V1 using the integrated led (pin2) I was only able to install the "All Clusters App for M5Stack ESP32 Basic Core" on my esp32, but it is not connecting.

Hello @saue0,

You should use **Kconfig** to select **ESP32-DevKitC**:
Check [lighting-app/esp32/main/Kconfig](https://github.com/project-chip/connectedhomeip/blob/master/examples/lighting-app/esp32/main/Kconfig.projbuild#L31)

```
            Note that the "ESP32-DevKitC" choice is compatible with a number of clone devices
            available from third-party manufacturers.

        config DEVICE_TYPE_ESP32_DEVKITC
            bool "ESP32-DevKitC"
            depends on IDF_TARGET_ESP32
```
LED_GPIO PIN here:			
https://github.com/project-chip/connectedhomeip/blob/master/examples/lighting-app/esp32/main/Kconfig.projbuild#L75
```
    config LED_GPIO
        int "LED GPIO number"
        range 0 48
        default 26 if DEVICE_TYPE_ESP32_WROVER_KIT
        default 8 if DEVICE_TYPE_ESP32_C3_DEVKITM || DEVICE_TYPE_ESP32H2_DEVKITC
        default 48 if DEVICE_TYPE_ESP32_S3_DEVKITM
        default 5
```

Regards

**Comment by Hedda** [comment]
Suggest that you maybe try to convince [Blakadder](https://github.com/blakadder) to start a Matter Device Database similar to his Zigbee Compatibility Device Repository and his Tasmota Device Templates Repository:

https://zigbee.blakadder.com

https://github.com/blakadder/zigbee

https://templates.blakadder.com/

https://github.com/blakadder/templates

PS: Blakadder is an active member and popular device hacker and blogger among Zigbee2MQTT, Tasmota, and Home Assistant communities:

https://blakadder.com

https://twitter.com/blakadder_

https://www.youtube.com/channel/UC7dFh9C1DHFJNTz2BFueO7w

**Comment by Hedda** [comment]
Nanoleaf has now announced that they will roll out Matter firmware in April of 2023 for the [Nanoleaf Essentials](https://nanoleaf.me/en-US/products/essentials/) line/series (based on Thread):

https://press.nanoleaf.me/2023/03/20/nanoleafs-essentials-matter-lights/

* https://nanoleaf.me/en-US/products/essentials/bulbs/?category=A19-E26&standard=matter&size=each
* https://nanoleaf.me/en-US/products/essentials/lightstrips/?category=lightstrips&standard=matter&pack=smarter-kit&size=2

Also, EVE announced their Matter firmware for Energy, Motion, Door & Window (Thread) devices will be out of Beta on 17th of April  2023:

https://www.evehome.com/en/upgrade-to-matter

* Eve Energy (Australia, Europe, U.K., U.S. & Canada)
  * https://www.evehome.com/en-us/identify-your-eve-accessory?product=eve-energy
* Eve Door & Window
  *  https://www.evehome.com/en-us/identify-your-eve-accessory?product=eve-door-window
* Eve Motion
  * https://www.evehome.com/en-us/identify-your-eve-accessory?product=eve-motion
